Write a _SUCCESS file to the relatedness Parquet output directory

lgruen opened the pull request:
Note: requires a fairly recent Google Cloud SDK (`gcloud`) installation.

jkgoodrich reviewed (approved):
This looks good to me and it worked. One thing is that we are still mostly using python 3.8. so I needed to change the `action=argparse.BooleanOptionalAction` to `action='store_true'` in `cloud_batch_submit.py`. Thank you so much for adding!!

lgruen commented:
> This looks good to me and it worked. One thing is that we are still mostly using python 3.8. so I needed to change the `action=argparse.BooleanOptionalAction` to `action='store_true'` in `cloud_batch_submit.py`. Thank you so much for adding!!

Great! I've changed the `action` upstream and updated the `cuKING` tag in this PR.

Generally I'd highly recommend switching to Python 3.10 (or now 3.11, which got released recently as is significantly faster), as there are lots of great new features. If Dataproc is holding you back, we've got a workaround for that too (although it's a bit of a hack).
